src/modal/__tests__/modal.test.js

```diff
@@ -149,6 +149,32 @@ describe('Modal', () => {
     expect(body.style.overflow).toBe('');
   });
 
+  test('resets body scroll on multiple modals closing', () => {
+    const TwoModals = ({isOpen1 = false, isOpen2 = false}) => (
+      <>
+        <Modal isOpen={isOpen1}>Modal 1</Modal>
+        <Modal isOpen={isOpen2}>Modal 2</Modal>,
+      </>
+    );
+
+    wrapper = mount(<TwoModals />);
+
+    const body = ((document.body: any): HTMLBodyElement);
+    expect(body.style.overflow).toBe('');
+
+    wrapper.setProps({isOpen1: true, isOpen2: false});
+    expect(body.style.overflow).toBe('hidden');
+
+    wrapper.setProps({isOpen1: true, isOpen2: true});
+    expect(body.style.overflow).toBe('hidden');
+
+    wrapper.setProps({isOpen1: false, isOpen2: true});
```

Comment:
Can you write a test that verifies closing the modals in the opposite order? It looks like that case is handled fine in the site preview, but I'd like to ensure we don't cause a regression.

Reply:
Sure thing. 
I don't think this will happen for the ESC case, since event listeners (keyup) get run in the order they're added, so the bottom modal should always unmount first.

Reply:
Added an additional test